arangod/Graph/ShortestPathOptions.cpp

```diff
@@ -212,18 +215,41 @@ auto ShortestPathOptions::estimateDepth() const noexcept -> uint64_t {
   return 7;
 }
 
+auto ShortestPathOptions::setDefaultWeight(double weight) -> void {
+  if (weight < 0.) {
```

Comment:
```suggestion
  if (weight <= 0.) {
```

Reply:
TY. will apply in separate commit as I am lazy and want to cherry-pick in a single commit :)

Reply:
reverted